controllers/metal3machine_controller_integration_test.go

```diff
@@ -579,6 +579,16 @@ var _ = Describe("Reconcile metal3machine", func() {
 					newMetal3Cluster(metal3ClusterName, nil, nil, nil, nil, false),
 					newBareMetalHost(nil, nil, nil, false),
 				},
+				TargetObjects: []runtime.Object{
+					&corev1.Node{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "bmh-0",
```

Comment:
I'm a bit confused by the name here. This is a Node, right? Is the name to indicate that it is using the BMH with the same name?

Reply:
Thanks. 
changed it to node-0, node-1 ...